packages/row/src/css/index.js

```diff
@@ -67,7 +55,12 @@ export default {
     background: transparentize(0.5, core.colors.black),
     transition: `opacity ${core.motion.speedNormal}`,
     pointerEvents: 'none',
-    opacity: 0
+    opacity: 0,
+    color: core.colors.white,
```

Comment:
this might be problematic. i'm using it to set the icon color - is this a breaking change?

Reply:
Which icon?  

If it wasn't being set before, perhaps people are setting it themselves.  And if they are, where would they be doing that?  And once we determine that, would this override what they're doing?  And if so, is that bad, since maybe we should have been doing it all along (if we aren't already, still not sure)?

So, I'm still not sure. :)

Reply:
the playicon that was in the story. if people are setting it themselves i don't think this will break anything since that would most likely be more specific. and yes i think this way is better

Reply:
If it's the playicon, I'd leave out the style.  The Card spec doesn't define what should be in the overlay where the play icon often is.

Reply:
will do